File: sdkwork-app-sdk-go/types/payment_status_vo.go
package types

// 支付状态响应
type PaymentStatusVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	PaymentOrderId string `json:"paymentOrderId"`
	MerchantOrderId string `json:"merchantOrderId"`
	Status string `json:"status"`
	StatusName string `json:"statusName"`
	Amount string `json:"amount"`
	PayTime int `json:"payTime"`
	PaymentMethod string `json:"paymentMethod"`
	PaymentId int `json:"paymentId"`
	PaymentSn string `json:"paymentSn"`
	OrderId int `json:"orderId"`
	PaymentProvider string `json:"paymentProvider"`
	TransactionId string `json:"transactionId"`
	OutTradeNo string `json:"outTradeNo"`
	SuccessTime string `json:"successTime"`
}

File: sdkwork-app-sdk-csharp/Models/PaymentStatusVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class PaymentStatusVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? PaymentOrderId { get; set; }
        public string? MerchantOrderId { get; set; }
        public string? Status { get; set; }
        public string? StatusName { get; set; }
        public string? Amount { get; set; }
        public int? PayTime { get; set; }
        public string? PaymentMethod { get; set; }
        public int? PaymentId { get; set; }
        public string? PaymentSn { get; set; }
        public int? OrderId { get; set; }
        public string? PaymentProvider { get; set; }
        public string? TransactionId { get; set; }
        public string? OutTradeNo { get; set; }
        public string? SuccessTime { get; set; }
    }
}
